src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleClick);
+  }
+
+  componentWillUnmount() {
+    setTimeout(() => {
+      document.removeEventListener('keyup', this.handleClick);
+    }, 3000);
+  }
+
+  handleClick = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: `The last prerssed key is [${event.key}]` });
+  };
+
+  render() {
+    const { pressedKey } = this.state;
+
+    return (
+      <div className="App">
+        <p>
+          {pressedKey}
```

Comment:
And here you can check if any key has been pressed and render a required message:
```
{pressedKey ? `The last pressed key is [${pressedKey}]` : 'Nothing was pressed yet'}
```

Reply:
Thanks)